functions.php

```diff
@@ -667,56 +823,70 @@ function capsule_wp_editor_warning() {
 }
 </style>
 <div class="capsule-editor-warning">
-	<h3><?php _e('Whoa Cowboy!', 'capsule'); ?></h3>
-	<p><?php printf(__('<b>Capsule is designed for front-end editing only.</b><br />Changes to projects, tags, etc. here will be overwritten when this post is edited on the front-end.<br /><a href="%s">Let\'s head back over there.</a>', 'capsule'), esc_url(home_url())); ?>
-	<p class="bypass"><?php _e('Ok, ok - I get it. <a href="#">Let me in anyway</a>.', 'capsule'); ?>
+	<h3><?php esc_html_e( 'Whoa Cowboy!', 'capsule' ); ?></h3>
+	<p>
```

Comment:
The markup is invalid. There are no closing tags for the paragraphs and the text in not indented correctly.